src/platform/packages/shared/kbn-scout/src/playwright/fixtures/scope/worker/apis/fleet/index.ts

```diff
@@ -9,44 +9,362 @@
 
 import type { KbnClient, ScoutLogger } from '../../../../../../common';
 import { measurePerformanceAsync } from '../../../../../../common';
+import type {
+  AgentPolicyCreateBody,
+  BulkGetBody,
+  FleetOutputBody,
+  FleetServerHostCreateBody,
+  AgentPolicyUpdateBody,
+} from './data_helper';
 
 export interface FleetApiService {
   integration: {
-    install: (name: string) => Promise<void>;
-    delete: (name: string) => Promise<void>;
+    install: (name: string) => Promise<any>;
+    delete: (name: string) => Promise<number>;
+  };
+  agent_policies: {
+    get: (queryParams?: Record<string, any>) => Promise<any>;
+    create: (
+      policyName: string,
+      policyNamespace: string,
+      sysMonitoring?: boolean,
+      params?: AgentPolicyCreateBody
+    ) => Promise<any>;
+    update: (
+      policyName: string,
+      policyNamespace: string,
+      agentPolicyId: string,
+      params?: AgentPolicyUpdateBody,
+      queryParams?: Record<string, string>
+    ) => Promise<any>;
+    bulkGet: (
+      bulkGetIds: string[],
+      params?: BulkGetBody,
+      queryParams?: Record<string, string>
+    ) => Promise<any>;
+    delete: (id: string, isForceSet?: boolean) => Promise<number>;
+  };
+  outputs: {
+    getOutputs: () => Promise<any>;
+    getOutput: (id: string) => Promise<any>;
+    create: (
+      outputName: string,
+      outputHosts: string[],
+      outputType: string,
+      params?: FleetOutputBody
+    ) => Promise<any>;
+    delete: (outputId: string) => Promise<number>;
+  };
+  server_hosts: {
+    get: () => Promise<any>;
+    create: (
+      hostName: string,
+      hostUrls: string[],
+      params?: FleetServerHostCreateBody
+    ) => Promise<any>;
+    delete: (id: string) => Promise<number>;
+  };
+  agent: {
+    setup: () => Promise<any>;
+    get: (queryParams: Record<string, any>) => Promise<any>;
+    delete: (agentId: string) => Promise<number>;
   };
 }
 
 export const getFleetApiHelper = (log: ScoutLogger, kbnClient: KbnClient): FleetApiService => {
   return {
     integration: {
       install: async (name: string) => {
-        await measurePerformanceAsync(log, `fleetApi.integration.install [${name}]`, async () => {
+        return await measurePerformanceAsync(
+          log,
+          `fleetApi.integration.install [${name}]`,
+          async () => {
+            return await kbnClient.request({
+              method: 'POST',
+              path: `/api/fleet/epm/custom_integrations`,
```

Comment:
Question: I [see](https://www.elastic.co/docs/api/doc/kibana/operation/operation-post-fleet-epm-custom-integrations) the `spaceId` is part of the URL. Should we extend the helper to support this parameter? We're doing it in the Cases API helper.